src/Microsoft.SqlTools.SqlCore/Localization/sr.strings

```diff
@@ -0,0 +1,432 @@
+﻿# String resource file
```

Comment:
I don't see the servicelayer project having these removed, are they still needed there or something?

And did you verify that all these strings are needed?